app/dcs/third-party-tools/veeam/page.md

```diff
@@ -193,3 +195,32 @@ Credentials window:
 1.  Review the information in the **Summary** window. Go back and make any necessary edits using the **Previous** button.
 
 2.  Select **Finish** and exit the wizard.
+
+### Job Configuration
+
+New Backup Job wizard to configure the backup job. Follow the steps below to complete the integration. This is an important step to optimize cost and performance of your backups. For more detail please see the following [Object Storage Considerations Veeam.](https://community.veeam.com/blogs-and-podcasts-57/sobr-veeam-capacity-tier-calculations-and-considerations-in-v11-2548)
+
+1. Launch the New Object Repository wizard:
+
+2. Work your way though the configuration and open **Advanced** on the [Storage Selection Step](https://helpcenter.veeam.com/docs/backup/vsphere/backup_job_storage_vm.html)
+
+![](https://link.storjshare.io/raw/jxujkau67l6nxg3yf5e3annjtdla/documentationimages/veeam1.png)
```

Comment:
if you add 3 spaces and indent the content related to the bullet point it'll render the html better for content readers and generally does better with SEO.

Example of what I mean https://github.com/storj/docs/blob/main/app/node/get-started/install-node-software/qnap-storage-node-app/page.md?plain=1#L30-L32